### PR TITLE
Transifex 2017 12 19

### PR DESCRIPTION
**Pull request by frederoni**
Pulled in the latest translations from Transifex

@1ec5 @bsudekum @JThramer 👀

### CODE REVIEW

File: MapboxNavigation/Resources/nl.lproj/Localizable.strings [hunk 1]
```diff
@@ -44,7 +44,7 @@
 "RESUME" = "Doorgaan";
 
 /* The text of a banner that appears during turn-by-turn navigation when route simulation is enabled. */
-"USER_IN_SIMULATION_MODE" = "Navigatie Simuleren";
+"USER_IN_SIMULATION_MODE" = "Simulating Navigation at %d×";
```
Comment (JThramer) [inline MapboxNavigation/Resources/nl.lproj/Localizable.strings]: Shouldn't this be `"Navigatie Simuleren op %dx"`?

Comment (frederoni) [reply MapboxNavigation/Resources/nl.lproj/Localizable.strings]: It should but this string has not yet been translated in Transifex.

File: MapboxNavigation/Resources/sv.lproj/Localizable.strings [hunk 2]
```diff
@@ -44,7 +44,7 @@
 "RESUME" = "Återuppta";
 
 /* The text of a banner that appears during turn-by-turn navigation when route simulation is enabled. */
-"USER_IN_SIMULATION_MODE" = "Simulera navigation";
+"USER_IN_SIMULATION_MODE" = "Simulerar Navigation  %d×";
```
Comment (JThramer) [inline MapboxNavigation/Resources/sv.lproj/Localizable.strings]: I'm nowhere near proficient in Swedish, but it kind of looks like a preposition is missing here?

Comment (frederoni) [reply MapboxNavigation/Resources/sv.lproj/Localizable.strings]: Good catch

**Comment by 1ec5** [comment]
Bulgarian is also ready to be added. Not sure why the scripts didn’t pick it up for this PR.

**Pull request merged**